server/channels/app/notification_push.go

```diff
@@ -108,7 +108,7 @@ func (a *App) sendPushNotificationToAllSessions(rctx request.CTX, msg *model.Pus
 	if rejectionReason != "" {
 		// Notifications rejected by a plugin should not be considered errors
 		// This is likely normal operation so no need for metrics here
-		a.NotificationsLog().Debug("Notification rejected by plugin",
+		a.Log().LogM(mlog.MlvlNotificationDebug, "Notification rejected by plugin",
```

Comment:
NOTE TO SELF: Check if `rctx` can be used here.

Reply:
I've migrated the code to use the corresponding `request.CTX` logger when ever it's possible: https://github.com/mattermost/mattermost/pull/33473/commits/6e3fe57a5ec5a536347d633f0935e8a5b0eb8391